scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerPage.java

```diff
@@ -442,39 +454,39 @@ public void onClick(ClickEvent event) {
             }
         });
 
-        schedStartButton = new MenuItem("Start");
+        schedStartButton = new MenuItem(START);
         schedStartButton.setIcon(SchedulerImages.instance.scheduler_start_16().getSafeUri().asString());
         schedStartButton.addClickHandler(new com.smartgwt.client.widgets.menu.events.ClickHandler() {
             public void onClick(MenuItemClickEvent event) {
                 SchedulerPage.this.controller.startScheduler();
             }
         });
 
-        schedStopButton = new MenuItem("Stop");
+        schedStopButton = new MenuItem(STOP);
         schedStopButton.setIcon(SchedulerImages.instance.scheduler_stop_16().getSafeUri().asString());
         schedStopButton.addClickHandler(new com.smartgwt.client.widgets.menu.events.ClickHandler() {
             public void onClick(MenuItemClickEvent event) {
                 SchedulerPage.this.controller.stopScheduler();
             }
         });
 
-        schedFreezeButton = new MenuItem("Freeze");
+        schedFreezeButton = new MenuItem(FREEZE);
         schedFreezeButton.setIcon(SchedulerImages.instance.scheduler_freeze_16().getSafeUri().asString());
         schedFreezeButton.addClickHandler(new com.smartgwt.client.widgets.menu.events.ClickHandler() {
             public void onClick(MenuItemClickEvent event) {
                 SchedulerPage.this.controller.freezeScheduler();
             }
         });
 
-        schedResumeButton = new MenuItem("Resume");
+        schedResumeButton = new MenuItem(RESUME);
         schedResumeButton.setIcon(SchedulerImages.instance.scheduler_resume_16().getSafeUri().asString());
         schedResumeButton.addClickHandler(new com.smartgwt.client.widgets.menu.events.ClickHandler() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)